Podfile

```diff
@@ -154,15 +154,11 @@ post_install do |installer|
   # Fix a code signing issue in Xcode 14 beta.
   # This solution is suggested here: https://github.com/CocoaPods/CocoaPods/issues/11402#issuecomment-1189861270
   # ====================================
-  #
-  # TODO: fix the linting issue if this workaround is still needed in Xcode 14 GM.
```

Comment:
Not sure if the comment is still valid. There is another one about Xcode 14 beta above. Since we are ultimately going to remove `Podifle`, I decided not to touch the code itself.